apps/lambda/greeting-sam.yaml

```diff
@@ -9,20 +9,18 @@ Resources:
       Handler: org.aws.samples.compute.greeting.GreetingHandler::handleRequest
       Runtime: java8
       CodeUri: ../../../microservices-greeting/target/greeting.zip
-      Description: >-
-        Amazon API Gateway.
       MemorySize: 1024
       Timeout: 20
       Tracing: Active
-      Policies:
```

Comment:
Are these policies no longer required? How would the `greeting` service put value to Parameter Store?

Reply:
This is more like an accident when I copy the canary setting into the template.